travel_platform/izi_travel_client.py

```diff
@@ -236,22 +237,23 @@ def get_museum_detail_with_audio(self, museum_uuid):
         prepared_params = {"includes": "download,city",
                            "except": "publisher,children"}
         params = self._prepare_params(**prepared_params)
-        museum_data = self._make_request(url, **params)
+        response = self._make_request(url, **params)
         detail_museum_info = {}
 
-        if museum_data:
+        if response:
 
-            content_provider = museum_data[0].get("content_provider", {})
+            museum_data = response.pop(0)
```

Comment:
Response is always a list, containing one object if we get one object or multiple objects if our request aimed at getting list of objects.